.circleci/config.yml

```diff
@@ -66,6 +66,14 @@ jobs:
             for n in {1..5}; do ../dockertest.sh $list && break; done
           no_output_timeout: 15m
 
+      - run: # test ./rueidisprob/go.mod
+          command: |
+            cd $CIRCLE_WORKING_DIRECTORY/rueidisprob
+            list=$(go list ./... | circleci tests split --split-by=timings)
+            echo "Test Packages: $list"
+            for n in {1..5}; do ../dockertest.sh $list && break; done
```

Comment:
I found that test failures may be caused by not yet initialized redis cluster. Adding a `sleep 5` after `docker-compose up -d` in the `dockertest.sh` could be helpful.

Reply:
[cfcfad5](https://github.com/redis/rueidis/pull/493/commits/cfcfad548c2dfaa251dec08cc18c711352a7f87d)

Yes. some time is needed to cluster configuration. deterministic approach is required, later we can do it.